tests/solvers/vof_subequations_interface_01.cc

```diff
@@ -59,6 +59,8 @@ test()
   solver_parameters.multiphysics.vof_parameters.surface_tension_force.enable =
     true;
 
+  std::cout << solver_parameters.multiphysics.VOF << std::endl;
```

Comment:
We should always output to the dealii log in the unit tests and not in the std::cout :)